src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/ClaimStore.java

```diff
@@ -45,7 +45,7 @@ public Claim saveClaim(ClaimData claimData) {
     }
 
     public Claim saveClaim(ClaimData claimData, String submitterId, LocalDate responseDeadline) {
-        logger.info(String.format("Saving claim: %s", claimData.getExternalId()));
+        logger.debug(String.format("Saving claim: %s", claimData.getExternalId()));
```

Comment:
SLF4J loggers already support placeholders, there is no need to use `String.format`. Please use
```
logger.debug("Saving claim: {}", claimData.getExternalId()));
```
Here and in all other places in this PR.